charts/opa-kube-mgmt/templates/deployment.yaml

```diff
@@ -170,6 +170,8 @@ spec:
         - name: mgmt
           image: {{ include "opa.mgmt.image" . }}
           imagePullPolicy: {{ .Values.mgmt.image.pullPolicy }}
+          startupProbe:
+{{ toYaml .Values.mgmt.startupProbe | indent 12 }}
```

Comment:
It's better to use `nindent`, so you can align this block properly with `startupProbe`.